portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java

```diff
@@ -903,9 +903,7 @@ else if (StringUtil.equalsIgnoreCase(
 		synchronized (this) {
 			portletActions = _getPortletMimeTypeActions(name, portlet);
 
-			if (!name.equals(PortletKeys.PORTAL)) {
-				_checkPortletActions(portlet, portletActions);
-			}
+			_checkPortletActions(portlet, portletActions);
```

Comment:
@dantewang Could you please describe the reason in more details? Why it won't be invoked?

Tina.